ConsoleApp/Engine.cs

```diff
@@ -0,0 +1,20 @@
+namespace Consoleapp;
+
+public enum EngineType
+{
+    inline,
```

Comment:
code convention violated here
should be
```suggestion
    Inline,
```